webview/src/shared/components/dragDrop/Draggable.tsx

```diff
@@ -26,16 +24,14 @@ export const Draggable: React.FC<DraggableProps> = ({
   disabled,
   dropTarget,
   onDrop,
-  onDragOver,
+  onDragEnter,
   onDragStart
   // eslint-disable-next-line sonarjs/cognitive-complexity
```

Comment:
[Q] Is this still needed?

Reply:
from the looks probably but thought I would check

Reply:
It passes sonar tests but not codeclimate. I checked.